### ISSUE TITLE
Plugins aren't bundled

### ISSUE THREAD

**Issue by shannon** [systemjs/plugin-babel#55]
It seems that my babel plugins aren't bundled unless I add them explicitly to the bundle command.

For example:

`jspm bundle myModule + babel-plugin-transform-decorators-legacy bundle.js`

Is this working as intended?

**Comment by guybedford** [comment]
Yes Babel plugins aren't bundled by default as the expectation is that they shouldn't be necessary in production.

**Comment by shannon** [comment]
Ok, I thought it might be something like that but I'm not quite sure how to get to a production state that doesn't require the plugins. I thought bundling was to get it to that state but maybe I'm missing something. I'll go back to the docs.

Thanks :-)

**Comment by shannon** [comment]
Ok I think I understand your comment now that I have gone back to it. If I bundle the application the plugins are no longer necessary because it's already transpiled. 

I figured out what I was doing wrong. I missed one of my own modules in the bundle command so then it was having to go fetch the babel plugins to transpile it. I was confused because I saw a bunch of requests going out for the babel plugins and I thought I had bundled everything.
